script.js

```diff
@@ -0,0 +1,165 @@
+const toDoForm = document.querySelector("form");
+const toDoInput = toDoForm.querySelector(".input-text");
+const toDoButton = toDoForm.querySelector(".input-button");
+
+const TODO_DB = "todo";
+let toDoList = [];
+
+// localStorage 모델: 웹이 로드 될 때 기존 투두리스트를 가져오기 위함
+const localStorageModel = {
+    // localStorage에 저장
+    save: () => {
+        localStorage.setItem(TODO_DB, JSON.stringify(toDoList));
+    },
+
+    // localStorage에서 로드
+    load: () => {
+        const loaded = localStorage.getItem(TODO_DB);
+        if (loaded !== null) {
+            const parsed = JSON.parse(loaded);
+            toDoList = parsed;
+        }
+    }
+}
+
+// List 모델
+const list = {
```

Comment:
위에서 `localStorageModel`으로 모델을 네이밍했다면, 여기서도 일관성을 지키기 위해 다음과 같이 이름을 바꾸면 어떨까 싶네요.

```suggestion
const listModel = {
```

또, 저의 개인적인 생각으로는 Model들은 `UpperCamelCase`를 쓰는 것이 더 좋아보이네요.
```suggestion
const ListModel = {
```

Reply:
반영했습니다~ 왜 이생각을 못 했을까요! ㅎㅎ 따로 파일을 만들 생각도했지만 파일이 너무 많아질까봐 한 파일 내에 냅두었는데 `UpperCamelCase`로 구분짓는다는 걸 깜박했네요..